marlowe-playground-client/src/Simulation/State.purs

```diff
@@ -236,9 +272,18 @@ emptyMarloweStateWithSlot sn =
   , editorErrors: mempty
   , editorWarnings: mempty
   , holes: mempty
-  , executionState: Just $ emptyExecutionStateWithSlot sn
+  , executionState: emptyExecutionStateWithSlot sn
   }
 
+getAsMuchStateAP :: forall m t0. Bind m => MonadState { marloweState :: NonEmptyList MarloweState | t0 } m => m State
```

Comment:
I don't think you need the `Bind m =>` here as it is implied by `MonadState s m =>`